### PR TITLE
audiocodec/streaming: Rearrange directory of streaming

### PR DESCRIPTION
**Pull request by zhouxinhe**
1. Move player.* to media/streaming
2. Move rb.* and rbs.* to media/utils

@jarvisjang @Taejun-Kwon

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [3f56690de68e1f10f8542a9d999685fd17560fce] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [3f56690de68e1f10f8542a9d999685fd17560fce] - Code Rule Check Result:
framework/include/media/util/rbs.h:36: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/util/rbs.h:46: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/util/rbs.h:47: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/util/rbs.h:48: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/util/rbs.h:49: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/util/rbs.h:50: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/util/rbs.h:50: ERROR: [IDT_M_TAB] please, no space before tabs
framework/include/media/util/rbs.h:51: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/streaming/player.c:198: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxW)
framework/src/media/util/internal_defs.h:28: ERROR: [BRC_M_SMT] that open brace { should be on the previous line
framework/src/media/util/internal_defs.h:36: ERROR: [BRC_M_SMT] that open brace { should be on the previous line
framework/src/media/util/internal_defs.h:44: ERROR: [BRC_M_SMT] that open brace { should be on the previous line

**Comment by seinfra** [comment]
Target : [d8848305c6650aa1a9bac20e5e9aecfaae189b66] - Code Rule Check (C++) OK.

**Comment by Taejun-Kwon** [comment]
Please consider below.
external : basic codec code from stagefright.
framework/include/media : headers of open api (remove mediautils.h from here) 
framework/src/media/utils : mediautils.cpp / mediautils.h / rb.c / rb.h
...

**Comment by zhouxinhe** [comment]
framework/include/media/MediaUtils.h is now included in InputDataSource.h, 
It need to be also opened, so I can not move it to framework/src/media/utils.
@Taejun-Kwon 

It's possible to keep rb.h & rbs.h as internal APIs.

**Comment by Taejun-Kwon** [comment]
Ok I will modify Inputdatasource.h first, and then you can do this later.

**Comment by Taejun-Kwon** [comment]
#1767 merged, please check.

**Comment by zhouxinhe** [comment]
Ok, I will.

**Comment by seinfra** [comment]
Target : [1d4ca0bc9983e99c70a2ba03f2535b9c9a71c8f9] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1d4ca0bc9983e99c70a2ba03f2535b9c9a71c8f9] - Code Rule Check Result:
framework/src/media/streaming/player.c:198: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxW)
framework/src/media/utils/internal_defs.h:28: ERROR: [BRC_M_SMT] that open brace { should be on the previous line
framework/src/media/utils/internal_defs.h:36: ERROR: [BRC_M_SMT] that open brace { should be on the previous line
framework/src/media/utils/internal_defs.h:44: ERROR: [BRC_M_SMT] that open brace { should be on the previous line
framework/src/media/utils/rbs.h:37: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:47: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:48: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:49: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:50: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:51: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:51: ERROR: [IDT_M_TAB] please, no space before tabs
framework/src/media/utils/rbs.h:52: ERROR: [IDT_M_TAB] code indent should use tabs where possible

**Comment by seinfra** [comment]
Target : [f1ea9f7b712c38cee7c047025b6dbfed174e2f6e] - Code Rule Check Result:
framework/src/media/utils/rbs.h:37: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:47: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:48: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:49: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:50: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:51: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/media/utils/rbs.h:51: ERROR: [IDT_M_TAB] please, no space before tabs
framework/src/media/utils/rbs.h:52: ERROR: [IDT_M_TAB] code indent should use tabs where possible

**Comment by seinfra** [comment]
Target : [f1ea9f7b712c38cee7c047025b6dbfed174e2f6e] - Code Rule Check (C++) OK.

File: framework/src/media/streaming/player.c [hunk 1]
```diff
@@ -24,8 +24,8 @@
 #include <assert.h>
 #include <pthread.h>
 #include <debug.h>
-#include <audiocodec/streaming/player.h>
-#include "internal_defs.h"
+#include "player.h"
+#include "../utils/internal_defs.h"
```
Comment (Taejun-Kwon) [inline framework/src/media/streaming/player.c]: please check this. we can remove '../'

Comment (zhouxinhe) [reply framework/src/media/streaming/player.c]: I tried, building failed without `../` :
CC:  src/media/streaming/player.c
src/media/streaming/player.c:28:33: fatal error: utils/internal_defs.h: No such file or directory
 #include "utils/internal_defs.h"

**Comment by Taejun-Kwon** [comment]
Please resolve conflict

**Pull request merged**